Inputs.md

```diff
@@ -0,0 +1,83 @@
+This guide will describe how inputs from the user (keyboard, mouse, etc.) are routed through InputWritr and how the DeviceLayr Gamepad can be used on top of it.
+
+### Table of Contents
+1. [InputWritr](#inputwritr)
+2. [DeviceLayr](#devicelayr)
+
+# InputWritr
+
+InputWritr is a module that automates interactions with user inputs and events. 
+
+Events are the game's responses to user input.
+They allow the user to take control of some part of the game or have interaction with it.
+Events can be added and removed using `addEvent` and `removeEvent`.
+
+```typescript
+InputWritr.addEvent("onKeyDown", 37, keyDownRightFunction);
+InputWritr.removeEvent("onKeyDown", 37);
+```
+
+Triggers are what allow the user to cause events.
+They are user inputs that get mapped to the events.
+They are stored as both character codes (e.g. 37) and as a string representation of their inputs (e.g. "left").
+Events can be triggered by any number of inputs.
+
+Aliases are additional inputs that allow for an event to be triggered from multiple user sources.
+They can be added and removed using `addAliasValues` and `removeAliasValues`.
+
+```typescript
+InputWritr.addAliasValues("left", [30, 34, 35]);
+InputWritr.removeAliasValues("left", [30, 34, 35]);
+```
+
+```typescript
+{
```

Comment:
Since this is in an events/triggers section I wouldn't add other properties like Timestamp right?